playground/infrastructure/helper.py

```diff
@@ -13,10 +13,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
+import re
+
 from dataclasses import dataclass
 from typing import List
+from api.v1.api_pb2 import Sdk, SDK_JAVA, Status
 
-from api.v1.api_pb2 import Sdk, Status
+SUPPORTED_SDK = {'java': SDK_JAVA}
+PATTERN = re.compile(
+    'Beam-playground:\n {2} *name: \w+\n {2} *description: .+\n {2} *multifile: (true|false)\n {2} *categories:\n( {4} *- [\w\-]+\n)+')
```

Comment:
please use **b**eam-playground pattern according to TDD

Reply:
Changed.